README.md

```diff
@@ -1 +1,56 @@
-# bookstore
\ No newline at end of file
+# BookStore
+
+> The Bookstore is a website similar to the "Awesome Books" website built in the previous module. You will create an MVP version of it that allows you to: Display a list of books. Add a book Remove a selected book. By building this application, you will learn how to use React and Redux.
+
+## Built With
+
+- React
+- HTML and CSS
+- Javascript
+- React
+
+## Live Demo (if available)
+
+[not available]()
+
+## Getting Started
+
+**How to clone the project and get started**
+
+To Run the project
+
+```
+1. git clone https://github.com/IamYunusAli/bookstore.git
+2. cd bookstore
+3. npm i
+4. npm run start
+5. use http://localhost:3000/ on your browser
+```
+
+enjoy
+
+## Authors
+
+👤 **Author **
+
+- GitHub: [@IamYunusALi](https://github.com/iamyunusali)
+- Twitter: [@IamYunusAli](https://twitter.com/iamyunusali)
+- LinkedIn: [Yunus Ali](https://linkedin.com/in/iamyunusali)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse team and the mentors
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Your are following almost all requirements but please add an MIT.md file to your project from this [template](https://github.com/microverseinc/readme-template/blob/master/MIT.md) :blush: because this is your license file so please take it seriously :+1:
- [ ] After adding it don't forget to add your name, current year and app-type in MIT.md file because most of the students forget :thinking: